src/components/dialogs/create-study-dialog/importParametersSection.tsx

```diff
@@ -59,6 +59,7 @@ const ImportParametersSection: FunctionComponent = () => {
                         onChange={handleParamsChange}
                         variant="standard"
                         selectionWithDialog={(param: any) => param.possibleValues?.length > 10}
+                        disabledParams={['iidm.import.cgmes.cgm-with-subnetworks']} //disabled parameters list
```

Comment:
In my opinion, it is better to extract to an constant outside of component and place in the same file, do not hard-code directly in components.

Reply:
ok

Reply:
solved